src/TLuaInterpreter.cpp

```diff
@@ -10953,7 +10953,7 @@ int TLuaInterpreter::getSpecialExitsSwap(lua_State* L)
             it.next();
             int id_to = it.key();
             QString dir = it.value();
-            // lua_pushstring(L, dir.toLatin1().data());
+            // lua_pushstring(L, dir.toUtf8().constData());
```

Comment:
As it has been commented out (for a long time?) I'd just remove the line entirely...!